schemas/context/profilestitch.schema.json

```diff
@@ -31,6 +31,12 @@
           },
           "minItems": 1,
           "description": "Array of Identities. Condensed, normalized encapsulation of all end user identifiers."
+        },
+        "xdm:endUserIDs": {
```

Comment:
my big worry here is that when we have IdentityMap, we'll have three fields for representing identity here. I get that we'll deprecate the old ones, but just want to document it here on the review.